pkg/kn/commands/service/describe.go

```diff
@@ -202,10 +202,10 @@ func describe(w io.Writer, service *v1alpha1.Service, revisions []*revisionDesc,
 func writeService(dw printers.PrefixWriter, service *v1alpha1.Service) {
 	dw.WriteColsLn(printers.Level0, l("Name"), service.Name)
 	dw.WriteColsLn(printers.Level0, l("Namespace"), service.Namespace)
-	dw.WriteColsLn(printers.Level0, l("URL"), extractURL(service))
+	dw.WriteColsLn(printers.Level0, l("Route URL"), extractURL(service))
```

Comment:
IMO, lets keep this field intact (use `URL`).

Reply:
The only thing which bothers me a bit that it's assymetric as "Cluster URL" is-a "URL", too. So maybe "External URL" is better ? We could even skip the "URL" part from cluster its clear from the value that its a URL. E.g.

```
URL:      http://myservice.external.com
Cluster:  http://myservice.myns.service.cluster.local
```

(alternative would be to replace "URL:" with "External:" or "Route:")

Reply:
Some variations:

```
URL:      http://myservice.example.com
Internal: http://myservice.myns.service.cluster.local
```

Reply:
```
External: http://myservice.example.com
Internal: http://myservice.myns.service.cluster.local
```

Reply:
```
External URL: http://myservice.example.com
Cluster URL:  http://myservice.myns.service.cluster.local
```

Reply:
```
URL:         http://myservice.example.com
Cluster URL: http://myservice.myns.service.cluster.local
```

Reply:
How about:

```
External URL: http://myservice.example.com
Cluster URL:   http://myservice.myns.service.cluster.local
```

Reply:
Oh, you already have that option :DD:D:D

Reply:
A final suggestion, to be clear about URL but reduce line width (as this is normally the biggest constraint, as URLs can become really long):

```
URL:
  External: http://myservice.example.com
  Cluster:  http://myservice.myns.service.cluster.local
```

Reply:
So we have a draw between URL/Cluster and External URL/Cluster URL. Considering @bbrowning comment, that the 'external' URL is not always external, I updated the PR to the "URL/Cluster" combo.

In a followup PR I would suggest hiding the cluster URL behind `--verbose` as this is really only of interest in very rare circumstances (e.g. when used as a Destination in Eventing)

Reply:
logged here https://github.com/knative/client/issues/482